common/modules/gcp-external-backup/main.tf

```diff
@@ -27,18 +27,7 @@ resource "google_storage_bucket" "external-backup-store" {
   location = "${var.location}"
   project  = "${var.project_id}"
 
-  storage_class = "MULTI_REGIONAL"
-
-  lifecycle_rule = {
-    action = {
-      type          = "SetStorageClass"
-      storage_class = "COLDLINE"
-    }
-
-    condition = {
-      age = "${var.days_until_coldline}"
-    }
-  }
+  storage_class = "COLDLINE"
```

Comment:
What region is the backup bucket in? It isn't clear to me what will happen here based on [the docs](https://cloud.google.com/storage/docs/storage-classes#coldline):

> The geo-redundancy of Coldline Storage data is determined by the type of location in which it is stored: Coldline Storage data stored in multi-regional locations is redundant across multiple regions, providing higher availability than Coldline Storage data stored in regional locations.

Anyway, we want the backups to be in a different region than the snapshots e.g. in case we need to recover from Google having an outage in our primary region.

Reply:
I can see in the properties of the bucket:
```
Location: us (multiple regions in United States)
Default storage class: Coldline 
```

- https://console.cloud.google.com/storage/browser/gpii-backup-stg?project=gpii2test-common-stg
- https://console.cloud.google.com/storage/browser/gpii-backup-prd?project=gpii2test-common-stg